src/plugins/auto/auto_schedule.cpp

```diff
@@ -116,6 +189,10 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
     // loadContext[ACTUALDEVICE] is always enabled,
     // when there is CPU and there are more than two devices, loadContext[CPU] is enabled
     _loadContext[ACTUALDEVICE].isEnabled = true;
+    if (_autoSContext->_runtimeFallback) {
+        _loadContext[FALLBACKDEVICE].isEnabled = true;
+        _loadContext[FALLBACKDEVICE].usefeature = false;
```

Comment:
do you mean usefuture?
is it able to use future also for the fallback task

Reply:
Consider some scenarios where switching to fallbackdevice and reinfer failed, so it need _loadContext[FALLBACKDEVICE] load again, if we want use the _loadContext[i].task again, if use promise, _loadContext[i].task cannot be ran again.

Reply:
Replaced `usefeature ` with `usePromise`